src/ui/home/components/claimOverview.tsx

```diff
@@ -78,51 +61,36 @@ const styles = StyleSheet.create({
 
 export class ClaimOverview extends React.Component<Props, State> {
 
-  renderClaimCards = (category: any) : ReactNode => {
+  renderClaimCards = (category: string) : ReactNode => {
     const { openClaimDetails, claims } = this.props
-    const categoryClaims = claims.savedClaims[category]
+    const decoratedClaims: CategorizedClaims = claims.claims
+    const categoryClaims: DecoratedClaims[] = decoratedClaims[category] || []
 
-    return categoryClaims.map((claim: IClaimUI, i: number) => {
-      // TODO: handle multiLine claims e.g. address for later
-      if (claim.multiLine) {
-        return
-      } else {
+    return categoryClaims.map((claim: DecoratedClaims, index) => {
         return (
           <ClaimCard
-            key={ claim.claimField }
```

Comment:
You could add back a key on ClaimCard render in order to get rid of the warning; maybe use claim.displayName here.